Azure-Samples/chat-with-your-data-solution-accelerator
# fix: Type HTM upload issue fix on explore page under admin

Bug: 5527

Purpose
it is for uploading HTM file and in explore dropdown HTM history also should be visible

## Does this introduce a breaking change?



## How to Test
Deploy CWYD
On Admin page under ingest data upload .HTML file
Under Explore data section's "Explore data' dropdown check if that uploaded file is visible or not
.HTM should be visible


git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
